assets/vue/components/PageBlocked.vue

```diff
@@ -54,21 +65,39 @@ export default {
       incidents: null,
       groupFlavors: true,
       matchText: '',
+      groupNames: '',
       refreshUrl: '/app/api/blocked'
     };
   },
   computed: {
     matchedIncidents() {
+      let results = this.incidents;
       if (this.matchText) {
-        return this.incidents.filter(incident => {
+        results = this.incidents.filter(incident => {
           if (String(incident.incident.number).includes(this.matchText)) return true;
           for (const pack of incident.incident.packages) {
             if (pack.includes(this.matchText)) return true;
           }
           return false;
         });
       }
-      return this.incidents;
+      if (this.groupNames) {
+        const groupNamesList = this.groupNames.toLowerCase().split(',');
+        return results.filter(incident => {
+          for (const key of Object.keys(incident.update_results)) {
+            for (const groupName of Object.values(groupNamesList)) {
+              if (groupName.toLowerCase() === incident.update_results[key].name.toLowerCase()) return true;
```

Comment:
```suggestion
              if (groupName === incident.update_results[key].name.toLowerCase()) return true;
```
Considering this comes from `const groupNamesList = this.groupNames.toLowerCase().split(',');` the case should be low enough at this point :-)